db/note.php

```diff
@@ -24,31 +24,38 @@
  * @method void setTitle(string $value)
  * @method string getContent()
  * @method void setContent(string $value)
+ * @method boolean getFavorite()
+ * @method void setFavorite(boolean $value)
  * @package OCA\Notes\Db
  */
 class Note extends Entity {
 
     public $modified;
     public $title;
     public $content;
+    public $favorite = false;
 
     public function __construct() {
         $this->addType('modified', 'integer');
+        $this->addType('favorite', 'boolean');
     }
 
     /**
      * @param File $file
      * @return static
      */
-    public static function fromFile(File $file){
+    public static function fromFile(File $file, array $tags=[]){
```

Comment:
If there are no favorites, the `array` annotation results in an Internal Server Error.

![image](https://cloud.githubusercontent.com/assets/1267827/19533799/8d911304-9641-11e6-8abc-17ac97bb3bf7.png)